docs/gambit/mutation-verifier.md

```diff
@@ -36,96 +36,80 @@ Once you have updated your `certora-cli` installation using `pip` to get the rel
-for generating the mutants in this `.conf` file; you don't need to write
-  two separate configurations.
-
-### Required Keys for the JSON Configuration File:
-- `"project_directory"` : the directory containing the original Prover project on which to perform mutation testing
-- `"run_script"` : the bash script used to run verification on the original project, usually `project_directory/run.sh` or similar.
-  Gambit will pull the configuration for `certoraRun` from this shell script and verify each mutant using this configuration.
-- `"gambit"` : the JSON configuration element for invoking Gambit. May be a path to a gambit configuration file
-or the explicit JSON element contained therein.  See {ref}`gambit-config` for more information about the gambit configuration.
-The `solc` specific arguments (including the version of the compiler) should be provided here
-  even if they are present in the `run_script`.
-
-### Optional Keys for the JSON Configuration File
-- `"num_threads"` : the maximum number of threads to use for verification, as an integer
-- `"manual_mutations"` : optionally supplement the random mutant generation with your own manually-written mutants.
-Expects a JSON object whose keys are the paths to the original files and whose values are paths to directories containing
-manually-written mutants as `.sol` files.
-
-```{note}
-Any manual mutations files provided must follow the naming
-convention
-`OriginalFileName.<unique-name>.sol`, where `<unique-name>` is a string ID unique with respect to the other
-manual mutants (for example you might name them `OriginalFileName.m1.sol`, `OriginalFileName.m2.sol` and so on).
-```
-
-### Additional Optional Flags
-
-```{note}
-Gambit supports `--server`.
-
-However, Gambit currently has trouble with
-{ref}`--send_only` in the run scripts.
-If you have this flag, please remove it for now.
-Apologies for the temporary inconvenience!
+```json
+{
+  "filename" : "ProjectDir/C.sol",
+  "num_mutants": 5
+}
 ```
 
-- `"offline"` : run mutation testing without internet connection, skipping the UI output and other web functions.
-Expects a boolean and defaults to `false`.
-- `"staging"` : if your run script does not already have `--server staging`, you can also add it to Gambit.
-  Similar to the Prover, you can provide the
-  branch name for running mutant verification on `--prover_version`.
-Omitting this key will cause verification to run locally
-  (unless the run script has it).
-- `"production"` : if you instead want to run on the production environment you can provide the `--server production` flag. You can also add the name of a specific branch using `--prover_version`
-- `"use_certora_run_py"` : Use `certoraRun.py` rather than `certoraRun`. Expects a boolean and defaults to `false`.
-
-
-### Troubleshooting
+## CLI Options
+
+`certoraMutate` runs in two distinct modes: synchronous and asynchronous. Use the `--sync` flag to run the entire tool synchronously
```

Comment:
Maybe it's a good idea to motivate and tease future versions.
First, to say that for more complex projects, async mode makes more sense, as it could take a long time to process, while for short runs with few mutants, sync mode makes more sense.
And then say the async mode will in the future allow to be automatically notified when all the mutation testing tasks are complete